### PR TITLE
Merge Development to Main

### PR DESCRIPTION
**Pull request by netrajpatel**

### CODE REVIEW

File: packages/contentstack-bootstrap/package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "rimraf": "^2.7.1",
     "tmp": "^0.2.1",
     "ts-node": "^8.10.2",
-    "typescript": "^3.9.7"
```
Comment (shafeeqd959) [inline packages/contentstack-bootstrap/package.json:49]: version bump

File: packages/contentstack-export/src/lib/export/entries.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ exportEntries.prototype.start = function (credentialConfig) {
   client = stack.Client(config)
   addlogs(config, 'Starting entry migration', 'success')
   return new Promise(function (resolve, reject) {
-    locales = helper.readFile(localesFilePath)
+    locales = helper.readFile(localesFilePath) || {};
```
Comment (shafeeqd959) [inline packages/contentstack-export/src/lib/export/entries.js:51]: version bump

File: packages/contentstack-seed/package.json [hunk 3]
```diff
@@ -33,7 +33,7 @@
     "jest": "^26.6.1",
     "ts-jest": "^26.4.2",
     "ts-node": "^8.10.2",
-    "typescript": "^3.9.7"
+    "typescript": "^4.7.4"
```
Comment (shafeeqd959) [inline packages/contentstack-seed/package.json:36]: version bump

File: packages/contentstack/package.json [hunk 4]
```diff
@@ -8,10 +8,10 @@
   },
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
-    "@contentstack/cli-auth": "0.1.1-beta.2",
+    "@contentstack/cli-auth": "^0.1.1-beta.3",
     "@contentstack/cli-cm-bootstrap": "^1.0.7",
-    "@contentstack/cli-cm-bulk-publish": "0.1.1-beta.5",
```
Comment (shafeeqd959) [inline packages/contentstack/package.json:13]: version bump

**Review by aravindbuilt — APPROVED** [review]

**Pull request merged**